tests/data/generator/two_party_negotiation/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2020 fetchai Limited
```

Comment:
@5A11 please change generator so it only says `Copyright {year} {author}`, i.e. in this case `Copyright 2020 fetchai`

Reply:
Done